testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,77 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
```

Comment:
Aha. Have you used `newapkbuild` when creating abuild? This automatically fills Maintainer.

Maintainer is a person who’s responsible for maintaining the package. That person i.a. gets mails from http://pkgs.alpinelinux.org/ when someone flags the package as outdated or something. Maintainer can be anyone, not just Alpine core or trusted devs.